src/App.jsx

```diff
@@ -25,22 +26,25 @@ const App = () => {
     }, [isLoading])
 
     return (
-        <>
-            <nav>
+        <div>
+            <nav className="bg-red-100 w-screen">
                 <ul>
                     <li>
-                        <Link to="/">
-                            Home
+                        <Link
+                            to="/">
+                            <BsUiChecks className="text-[30px] fill-red-900"/>
```

Comment:
Better to use rem / em values as they are more flexible and can be more easily controlled on a global level

Reply:
Changed